src/sdltiles.cpp

```diff
@@ -4123,3 +4062,64 @@ HWND getWindowHandle()
 #endif
 
 #endif // TILES
+
+static window_dimensions get_window_dimensions( const catacurses::window &win,
```

Comment:
This need to be inside the TILES define because it uses fonts which is not available on curses.